lib/benchee/benchmark/measure/memory.ex

```diff
@@ -11,10 +11,19 @@ defmodule Benchee.Benchmark.Measure.Memory do
     ref = make_ref()
     Process.flag(:trap_exit, true)
     start_runner(fun, ref)
+    await_results({nil, nil}, ref)
+  end
 
+  def await_results({_, result}, ref) do
```

Comment:
If we always throw he first element of the tuple away, why do we have it at all?

Reply:
Good call - I'll update this.